report-pox-krypton.js

```diff
@@ -771,16 +779,59 @@ function process_burnchain_ops() {
     console.log(`${miner_key}/${c32.c32ToB58(miner_key)} ${miner.actual_win}/${miner.won}/${miner.mined} ${(miner.won / miner.mined * 100).toFixed(2)}% ${(miner.actual_win / actual_win_total * 100).toFixed(2)}% - ${miner.burned} - Th[${(miner.burned / miner.total_burn * 100).toFixed(2)}%] (${miner.burned / miner.mined})`)
     miner.average_burn = miner.burned / miner.mined
     miner.normalized_wins = miner.won / miner.average_burn
+    miner.burn_per_win = miner.burned / miner.won;
+    miner.profit = miner.won * 500 * 0.15 - (miner.burned / 100000000) * 13000;
```

Comment:
should use 1000 (I know the code has not been changed in the node, but that's the plan), and maybe the token values should be using environment variables?  The STX and BTC prices are moving fast.

Reply:
and regarding profit calculation, I think it would be helpful to also include the fee miners would need to pay on mainnet.  For example, depending on the service you use, today, you'd need to pay 176-222 satoshis per byte (the transaction is 352), so that's $8-10 per miner per block...  I would also make this an environment variable (sats/b)